package.json

```diff
@@ -50,6 +50,7 @@
   "devDependencies": {
     "babel-eslint": "^10.0.1",
     "eslint": "^5.9.0",
+    "is-promise": "^4.0.0",
```

Comment:
Do we really need to pull in a third-party library here?

Reply:
I want use `util.types.isPromise`, but it added in v10...